charts/ceph-csi-rbd/values.yaml

```diff
@@ -281,8 +281,9 @@ storageClass:
 
   # (optional) RBD image features, CSI creates image with image-format 2 CSI
   # RBD currently supports `layering`, `journaling`, `exclusive-lock`,
-  # `object-map`, `fast-diff`, `deep-flatten` features. If `journaling` is
-  # enabled, must enable `exclusive-lock` too.
+  # `object-map`, `fast-diff`, `deep-flatten` features.
+  # Refer https://docs.ceph.com/en/latest/rbd/rbd-config-ref/#image-features
```

Comment:
```suggestion
  # Note: Refer https://docs.ceph.com/en/latest/rbd/rbd-config-ref/#image-features
```

Reply:
we dont add note. As this is applicable if they use more image features, as the reference is given IMO should be fine. if the dependencies are not mentioned properly the volume creation will fail with the proper error message.